docs/naivebayes.md

```diff
@@ -0,0 +1,108 @@
+Naive Bayes Classifiers
+=======================
+
+The Naive Bayes classifiers is a simple probabilistic classification model based on Bayes Theorem. It is a simple, yet powerful classification model. Since Naive Bayes classifiers simply classifies sets of data by which class has the highest likelihood, Naive Bayes classifiers can use any dsitribution or model that outputs a probability.
```

Comment:
* "The Naive Bayes classifier", not classifiers
* It is a simple and extremely fast classification model. I don't know if powerful is the right word
* It's not about the highest likelihood, it's about likelihood * prior, like Bayes Rule. 
* "Naive Bayes classifiers can use any distribution (such as NormalDistribution) or model (such as HiddenMarkovModel) which has a probabilistic interpretation of the data."